password_security/__manifest__.py

```diff
@@ -1,29 +1,26 @@
+# pylint: disable=missing-docstring
 # Copyright 2015 LasLabs Inc.
+# Copyright 2018 Modoolar <info@modoolar.com>.
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 {
-
-    'name': 'Password Security',
+    "name": "Password Security",
     "summary": "Allow admin to set password security requirements.",
-    'version': '11.0.1.1.0',
-    'author':
-        "LasLabs, "
-        "Kaushal Prajapati, "
-        "Tecnativa, "
-        "Odoo Community Association (OCA)",
-    'category': 'Base',
-    'depends': [
-        'auth_crypt',
-        'auth_signup',
-    ],
-    "website": "https://laslabs.com",
+    "version": "12.0.1.1.0",
+    "author": "LasLabs, "
+    "Kaushal Prajapati, "
+    "Tecnativa, "
+    "Modoolar, "
+    "Odoo Community Association (OCA)",
+    "category": "Base",
+    "depends": ["auth_signup", "auth_password_policy_signup"],
```

Comment:
Please avoid all of these cosmetic changes. I see people changing from double to single quotes and viceversa all the time. Just don't touch if you don't need to please.